selfdrive/car/toyota/values.py

```diff
@@ -636,6 +636,7 @@ class CAR:
       b'\x018966342M5000\x00\x00\x00\x00',
       b'\x02896630ZQ3000\x00\x00\x00\x008966A4703000\x00\x00\x00\x00',
       b'\x02896630ZR2000\x00\x00\x00\x008966A4703000\x00\x00\x00\x00',
+      b'\x02896630ZT8000\x00\x00\x00\x008966A4703000\x00\x00\x00\x00,
```

Comment:
missing a closing quote

Reply:
Thank you for suggestion! Have fixed that.